bgcval2/analysis_compare.py

```diff
@@ -183,163 +181,14 @@ def timeseries_compare(jobs,
-        #                analysisKeys.append('Si')                       # WOA Siliate
-
-        #                analysisKeys.append('TotalAirSeaFluxCO2')          # work in progress
-        #                analysisKeys.append('AirSeaFlux')          # work in progress
-        #                analysisKeys.append('AirSeaFluxCO2')          # work in progress
-
-        #                analysisKeys.append('scvoltot')
-        #                analysisKeys.append('soga')
-        #                analysisKeys.append('thetaoga')
-        #                analysisKeys.append('scalarHeatContent')
-
-        #analysisKeys.append('ADRC_26N')                # AMOC 26N
-        #                analysisKeys.append('VerticalCurrent')          # Vertical Veloctity
-        #                analysisKeys.append('sossheig')                 # SSH
-        #                analysisKeys.append('NorthernTotalIceArea')     # North TotalIceArea
-        #                analysisKeys.append('SouthernTotalIceArea')     # South TotalIceArea
-        #                analysisKeys.append('TotalIceArea')     	#  TotalIceArea
-        #                analysisKeys.append('NorthernTotalIceExtent')   # work in progress
-        #                analysisKeys.append('SouthernTotalIceExtent')   # work in progress
-        #                analysisKeys.append('WeddelIceExent')   # work in progress
-        #                analysisKeys.append('TotalIceExtent')           # work in progress
-        #                analysisKeys.append('NorthernMIZArea')
-        #                analysisKeys.append('SouthernMIZArea')
-        #                analysisKeys.append('TotalMIZArea')
-
-        #                        analysisKeys.append('NorthernMIZArea')
-        #                        analysisKeys.append('SouthernMIZArea')
-        #                        analysisKeys.append('TotalMIZArea')
-        #                analysisKeys.append('NorthernMIZfraction')
-        #                analysisKeys.append('SouthernMIZfraction')
-        #                analysisKeys.append('TotalMIZfraction')
-
-        #		analysisKeys.append('FreshwaterFlux')		# Fresh water flux
-        #analysisKeys.append('GlobalMeanTemperature')
-#                analysisKeys.append('GlobalMeanSalinity')
-
-#                analysisKeys.append('HeatFlux')
-#                analysisKeys.append('TotalHeatFlux')
-
-#               	analysisKeys.append('quickSST')    		# Area Weighted Mean Surface Temperature
-#       	  	analysisKeys.append('TotalOMZVolume')           # Total Oxygen Minimum zone Volume
-#       	 	analysisKeys.append('OMZThickness')             # Oxygen Minimum Zone Thickness
-#        	analysisKeys.append('OMZMeanDepth')             # Oxygen Minimum Zone mean depth
-#		analysisKeys.append('O2')                       # WOA Oxygen
-#       	if bio ==False:return
-#       	if physics == True:return
+    if isinstance(suites, str):
+        suites = [suites, ]
```

Comment:
add an `else:
                  ValueError(f"Suites need to be a string, got {suites}")
                   sys.exit(1)`

Reply:
Actually, suites need to be a list! I'll amend it here.